index.js

```diff
@@ -1,39 +1,604 @@
-const readline = require('readline');
-const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+const readline = require("readline");
+const readlineInterface = readline.createInterface(
+  process.stdin,
+  process.stdout
+);
 
 function ask(questionText) {
   return new Promise((resolve, reject) => {
     readlineInterface.question(questionText, resolve);
   });
 }
 
-// remember the StateMachine lecture
-// https://bootcamp.burlingtoncodeacademy.com/lessons/cs/state-machines
+const mainStMessage = `182 Main St. You are standing on Main Street between Church and South Winooski.
```

Comment:
It's good that you're using const to declare these global variables since it will throw an error if anything tries to change them, but you should still be wary of global objects.